intelmq/conf/harmonization.conf

```diff
@@ -311,6 +311,10 @@
             "description": "The URL of a given abuse feed, where applicable",
             "type": "URL"
         },
+        "feed.accuracy": {
```

Comment:
@sebix in your opinion we should use the same procedure like the following?
https://github.com/certtools/intelmq/blob/master/intelmq/lib/message.py#L221